### PR TITLE
build: add deskflow devs to binaries copyright

### PR DESCRIPTION
**Pull request by sithlord48**
Have the entry for windows / mac binaries say 

© 2024 Deskflow Developers

### CODE REVIEW

File: res/dist/mac/bundle/Contents/Info.plist.in [hunk 1]
```diff
@@ -24,7 +24,8 @@
     <key>CFBundleVersion</key>
     <string>@DESKFLOW_VERSION@</string>
     <key>NSHumanReadableCopyright</key>
-    <string>© 2012-@DESKFLOW_BUILD_YEAR@ Symless Ltd</string>
+    <string>© 2024 Deskflow Developers
+  © 2012 - 2024 Symless Ltd.</string>
```
Comment (nbolton) [inline res/dist/mac/bundle/Contents/Info.plist.in:28]:
```suggestion
    <string>© 2024 Deskflow Developers, © 2012-2024 Symless Ltd.</string>
```

File: res/win/version.rc.in [hunk 2]
```diff
@@ -10,12 +10,12 @@ https://learn.microsoft.com/en-us/windows/win32/menurc/versioninfo-resource?redi
...
 
 #define VER_PRODUCTVERSION          @DESKFLOW_VERSION_MS_CSV@
-#define VER_PRODUCTVERSION_STR      "@DESKFLOW_VERSION_MS@\0"
+#define VER_PRODUCTVERSION_STR      "@DESKFLOW_VERSION@\0"
 
 #define VER_COMPANYNAME_STR         "@DESKFLOW_AUTHOR_NAME@\0"
 #define VER_FILEDESCRIPTION_STR     "@DESKFLOW_APP_NAME@\0"
 #define VER_INTERNALNAME_STR        "@DESKFLOW_APP_NAME@\0"
-#define VER_LEGALCOPYRIGHT_STR      "Copyright (C) Symless Ltd. @DESKFLOW_BUILD_YEAR@\0"
+ #define VER_LEGALCOPYRIGHT_STR      "2024 Deskflow Developers 2012 - 2024 Symless Ltd.\0"
```
Comment (nbolton) [inline res/win/version.rc.in:18]: Does © work here? If not, maybe (C)?
```suggestion
 #define VER_LEGALCOPYRIGHT_STR      "© 2024 Deskflow Developers, © 2012-2024 Symless Ltd.\0"
```

**Comment by sithlord48** [comment]
No this return and two spaces lines them up perfect on Mac oss dialog

**Comment by sithlord48** [comment]
Windows shows limited characters so adding (c) and copyright cut off the
text more

File: res/dist/mac/bundle/Contents/Info.plist.in (hunk 1 above)
Comment (sithlord48) [reply res/dist/mac/bundle/Contents/Info.plist.in:28]: No on mac os the return and spaces line them up correctly vertically

File: res/win/version.rc.in (hunk 2 above)
Comment (sithlord48) [reply res/win/version.rc.in:18]: Windows is very limited on the space it will not show more then one line and it does not allow users to resize..

Comment (nbolton) [reply res/win/version.rc.in:18]: In my suggestion, I dropped 2 chars and added 1... does it still wrap?

Comment (nbolton) [reply res/win/version.rc.in:18]: New idea...
```suggestion
#define VER_LEGALCOPYRIGHT_STR      "2024 Deskflow Developers, 2012-2024 Symless Ltd.\0"
```

Comment (sithlord48) [reply res/win/version.rc.in:18]: It does not wrap on windows it just cuts it off

Comment (nbolton) [reply res/win/version.rc.in]: In my 2nd suggestion I have removed 2 chars, and only added 1 char.

**Review by nbolton — APPROVED** [review]

**Pull request merged**